command_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package updater
+
+import (
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/keybase/go-logging"
+)
+
+func TestEmptyRunCommand(t *testing.T) {
+	log := logging.Logger{Module: "test"}
+	_, err := RunCommand("", nil, time.Second, log)
+	t.Logf("Error: %s", err)
+	if err == nil {
+		t.Fatal("Should have errored")
+	}
+}
+
+func TestInvalidRunCommand(t *testing.T) {
+	log := logging.Logger{Module: "test"}
+	_, err := RunCommand("invalidexecutable", nil, time.Second, log)
+	t.Logf("Error: %s", err)
+	if err == nil {
+		t.Fatal("Should have errored")
+	}
+}
+
+func TestRunCommandEcho(t *testing.T) {
+	log := logging.Logger{Module: "test"}
+	out, err := RunCommand("echo", []string{"arg1", "arg2"}, time.Second, log)
```

Comment:
so exec.Cmd CombinedOutput() is std library function which says it gives combined output... So I don't think we need to test that.